presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/ExpressionVerifier.java

```diff
@@ -235,4 +239,17 @@ protected Boolean visitSymbolReference(SymbolReference actual, Expression expect
         }
         return symbolAliases.get(((SymbolReference) expected).getName()).equals(actual);
     }
+
+    @Override
+    protected Boolean visitCoalesceExpression(CoalesceExpression actual, Expression expected)
```

Comment:
commit message does not say anything about this. I think it should be a separate commit.